locale/en-us.yaml

```diff
@@ -1573,19 +1573,23 @@ ppm_print_main:
     See "https://docs.activestate.com/platform/state/" for details.
 ppm_print_forward:
   other: |
-    Your command has been forwarded to `{{.V0}}`. 
-    In the future you can run `{{.V0}}` directly instead of running `{{.V1}}`, which will eventually disappear entirely.
+    Your command is being forwarded to `[BOLD][BLUE]{{.V0}}[/RESET]`.
+    In the future you can run `[BOLD][BLUE]{{.V0}}[/RESET]` directly instead of running `[BOLD][BLUE]{{.V1}}[/RESET]`, which will eventually disappear entirely.
+ppm_print_forward_after_convert:
+  other: |
+    Please navigate to your newly created project directory and run `[BOLD][BLUE]{{.V1}}[/RESET]` again.
+    You can also run `[BOLD][BLUE]{{.V0}}[/RESET]` directly instead of running `[BOLD][BLUE]{{.V1}}[/RESET]`, which will eventually disappear entirely.
 ppm_print_forward_failure:
   other: |
-    Your command has been forwarded to `{{.V0}}`. 
-    In the future you can run `{{.V0}}` directly instead of running `{{.V1}}`, which will eventually disappear entirely.
+    Your command is being forwarded to `[BOLD][BLUE]{{.V0}}[/RESET]`.
+    In the future you can run `[BOLD][BLUE]{{.V0}}[/RESET]` directly instead of running `[BOLD][BLUE]{{.V1}}[/RESET]`, which will eventually disappear entirely.
 
-    Note that `{{.V0}}` is not a direct analog for `{{.V1}}` and your command may have failed as a result. 
-    Please run `{{.V0}} --help` for further information on how to use it for your use-case.
+    Note that `[BOLD][BLUE]{{.V0}}[/RESET]` is not a direct analog for `[BOLD][BLUE]{{.V1}}[/RESET]` and your command may have failed as a result.
+    Please run `[BOLD][BLUE]{{.V0}} --help[/RESET]` for further information on how to use it for your use-case.
 ppm_print_redundant:
   other: |
-    ppm area and its subcommands have been made redundant thanks to State Tool's use of virtual environments. 
-    You can simply start managing your packages, run `{{.V0}} --help` for more information.
+    `ppm area` and its subcommands have been made redundant thanks to State Tool's use of virtual environments.
```

Comment:
nit: this one is on me, but `ppm area` should probably not be hard coded in this locale entry incase we want to use it again.

Reply:
I'd say the problem here is the locale entry key, not the value.